instrumentation/runtime-telemetry/runtime-telemetry-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java17/internal/garbagecollection/G1GarbageCollectionHandler.java

```diff
@@ -27,21 +27,20 @@ public final class G1GarbageCollectionHandler implements RecordedEventHandler {
           "G1 Young Generation",
           Constants.ATTR_GC_ACTION,
           Constants.END_OF_MINOR_GC);
-  private final LongHistogram histogram;
+  private final DoubleHistogram histogram;
 
   public G1GarbageCollectionHandler(Meter meter) {
     histogram =
         meter
             .histogramBuilder(Constants.METRIC_NAME_GC_DURATION)
             .setDescription(Constants.METRIC_DESCRIPTION_GC_DURATION)
-            .setUnit(Constants.MILLISECONDS)
-            .ofLongs()
+            .setUnit(Constants.SECONDS)
             .build();
   }
 
   @Override
   public void accept(RecordedEvent ev) {
-    histogram.record(ev.getLong(Constants.DURATION), ATTR);
+    histogram.record(ev.getDouble(Constants.DURATION), ATTR);
```

Comment:
@robsunday I'm writing up the release notes for the upcoming release and had a question here, does this need a conversion from milliseconds to seconds? thanks

Reply:
Yes, you are right - conversion should be added here and in all other modified handlers. Thanks for a great catch - I missed that. I'll update it.

Reply:
Hopefully resolved with https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12244